ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -168,3 +168,35 @@ keys:
                           - "mbytes"
                           - "packets"
                         default: "bytes"
+      copp_system_policy:
+        type: dict
+        description: Control-plane policy configuration.
+        keys:
+          enable:
+            type: bool
+            required: true
+            description: Enable control-plane policy.
+          classes:
+            type: list
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  required: true
+                shape:
+                  type: int
+                  description: Maximum rate limit.
+                  min: 0
+                  max: 10000000
+                bandwidth:
+                  type: int
+                  description: Minimum bandwidth.
+                  min: 0
+                  max: 10000000
+                rate_unit:
+                  type: str
+                  required: true
```

Comment:
I don't think this should be required, please check once.

Reply:
command for shape configuration is 'shape pps\kbps <value>'. So we must define rate unit.